### PR TITLE
Issue #12 Fix

### PR DESCRIPTION
**Pull request by PrakashDurlabhji**

### CODE REVIEW

**Comment by PrakashDurlabhji** [comment]
@luizrrodrigues can you please check now on safari?

**Comment by luizrrodrigues** [comment]
@PrakashDurlabhji Can you fix conflicts please?
Also, you able to run `npm num build` without any errors?
What version of nodejs you're using?

**Comment by PrakashDurlabhji** [comment]
@luizrrodrigues PR update, kindly check now

**Comment by PrakashDurlabhji** [comment]
@luizrrodrigues video is too fast, not able to predict what is happening,
Also I dont have mac on which i can test it.

trying to find out what can be done for mac such that other browsers should not affect.

**Comment by luizrrodrigues** [comment]
I'll record new one and test in Windows shortly

**Comment by PrakashDurlabhji** [comment]
@luizrrodrigues but it is working fine in chrome, firefox and IE correct?
I am confused what to do for Mac and how to test locally.

**Comment by luizrrodrigues** [comment]
@PrakashDurlabhji Yes, working in Chrome, Firefox, Edge.

File: src/components/TopNav/SubNav.module.scss [hunk 1]
```diff
@@ -42,7 +42,30 @@
...
+      height: 100%;
+      display: flex;
+      position: relative;
+      flex-direction: column;
+      -webkit-justify-content: center;
+      justify-content: center;
 
+      .indicator {
+        display: none;
+        position: absolute;
```
Comment (luizrrodrigues) [inline src/components/TopNav/SubNav.module.scss]:
```suggestion
```

File: src/components/TopNav/SubNav.module.scss [hunk 2]
```diff
@@ -42,7 +42,30 @@
...
+        height: 3px;
+        border-radius: 1.5px;
+        background: $gray-10;
+        -webkit-align-self: center;
+        align-self: center;
+      }
+
+      &:hover {
+        .indicator {
+          display: inline-block;
```
Comment (luizrrodrigues) [inline src/components/TopNav/SubNav.module.scss]:
```suggestion
          display: inline-block;
          position: absolute;
```

**Comment by PrakashDurlabhji** [comment]
@luizrrodrigues it is done

**Pull request merged**